lib/text_delta/attributes.ex

```diff
@@ -238,4 +240,21 @@ defmodule TextDelta.Attributes do
     |> Enum.filter(fn {key, _} -> not Map.has_key?(attrs_b, key) end)
     |> Enum.into(%{})
   end
+
+  def invalid_document_attributes?(attrs) when is_nil(attrs), do: true
+
+  def invalid_document_attributes?(attrs) do
+    attrs
+      |> Enum.find(&invalid_document_attribute?/1)
+      |> case do
+        nil -> false
+        _ -> true
+      end
+  end
+
+  def invalid_document_attribute?({_, %{ops: _} = attr}) do
```

Comment:
might be cleaner to have `def is_document_attribute_invalid?`, as right now its a bit hard to read (one letter diff :hiddapain:)

Reply:
also remove extra tuple from params

Reply:
i used `is_invalid_document_attribute`, because `is_document_attribute_invalid` suggests that the input param is supposed to be a `document_attribute` which is not the case, that ok?

Reply:
yes, thats gewd